src/userPresence/userPresence.ts

```diff
@@ -21,15 +21,11 @@ export const UserPresence = {
       lastFocusedTime,
       lastKeyStrokeTime,
       isFocused,
-      previousPresenceStatus,
```

Comment:
We don't need `previousPrsenceStatus` any more. Remove `previousPrsenceStatus` form global cache object and also remove code for setting it.